web-registry/src/pages/CreditClassDetails.tsx

```diff
@@ -115,215 +31,55 @@ function CreditClassDetails(): JSX.Element {
 }
 
 function CreditClassDetail({ isLandSteward }: CreditDetailsProps): JSX.Element {
-  const [modalIframeLink, setModalIframeLink] = useState<string>('');
-  const [isBuyerModalOpen, setBuyerModalOpen] = useState(false);
-  const [submitted, setSubmitted] = useState(false);
-  const styles = useStyles();
-  const navigate = useNavigate();
-
   let { creditClassId } = useParams();
+  const [ledgerClass, setLedgerClass] = useState<any>(undefined);
+  const [metadata, setMetadata] = useState<any>(undefined);
 
   const { data } = useAllCreditClassQuery({ client });
+  const { data: dbData } = useCreditClassByOnChainIdQuery({
+    variables: { onChainId: creditClassId?.toUpperCase() || '' },
+  });
+
+  useEffect(() => {
+    if (creditClassId) {
+      getClassInfo(creditClassId.toUpperCase()).then(res => {
```

Comment:
ditto

Reply:
moved toUpperCase up, to avoid repeating

Reply:
see above